mllib/src/test/scala/org/apache/spark/ml/linalg/VectorUDTSuite.scala

```diff
@@ -17,9 +17,19 @@
 
 package org.apache.spark.ml.linalg
 
+import scala.beans.BeanInfo
+
 import org.apache.spark.SparkFunSuite
+import org.apache.spark.sql.catalyst.JavaTypeInference
 import org.apache.spark.sql.types._
 
+@BeanInfo
+case class LabeledPoint(label: Double, features: Vector) {
+  override def toString: String = {
+    s"($label,$features)"
+  }
+}
```

Comment:
TODO, I'll remove this `LabeledPoint ` class in https://github.com/apache/spark/pull/12627